edk2toolext/tests/test_edk2_plat_build.py

```diff
@@ -0,0 +1,123 @@
+# @file test_edk2_plat_build.py
+# This contains unit tests for the edk2_plat_build
+##
+# Copyright (c) Microsoft Corporation
+#
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+import unittest
+from edk2toolext.invocables.edk2_platform_build import Edk2PlatformBuild
+import tempfile
+import sys
+import os
+import logging
+from importlib import reload
+from edk2toolext.environment import shell_environment
+
+class_info = '''
```

Comment:
Rather than take this approach to spontaneous file creation (since this has to be done for almost every test), could we instead move all these tests into a sub-directory of tests/ and include test files alongside them?

Reply:
Hmmm- it's an interesting idea. Have a minimum viable uefi tree rather than dynamically creating one.

Reply:
Alright- tried it out, let me know what you think?